capsules/calculator_object_speed/backend.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+from time import time
+import math
+
+import numpy as np
+
+from vcap import (
+    BaseBackend,
+    DETECTION_NODE_TYPE,
+    OPTION_TYPE)
+from .stream_state import StreamState, DetectionTimestamp
+
+
+class Backend(BaseBackend):
+
+    def process_frame(self, frame: np.ndarray,
+                      detection_node: DETECTION_NODE_TYPE,
+                      options: Dict[str, OPTION_TYPE],
+                      state: StreamState) -> DETECTION_NODE_TYPE:
+        capacity = options["cache_capacity"]
+        last = state.get(detection_node.track_id)
+        new = DetectionTimestamp(tstamp=time(),
+                                 coordinate=detection_node.bbox.center)
+        if last is not None:
+            # Calculate speed if it's not the first track
+            pixel_distance = math.hypot(
+                new.coordinate[0] - last.coordinate[0],
+                new.coordinate[1] - last.coordinate[1])
```

Comment:
Turns out `math.dist` is a thing:
```suggestion
            pixel_distance = math.dist(new.coordinate, last.coordinate)
```

Reply:
Wait, nvm. Python 3.8+